python/pipeline/reso.py

```diff
@@ -1051,12 +1051,12 @@ class CNMFBackground(dj.Part):
         activity            : longblob      # array (num_background_components x timesteps)
         """
 
-    def _make_tuples(self, key):
+    def make(self, key):
         # Create masks
         if key['segmentation_method'] == 1:  # manual
-            Segmentation.Manual()._make_tuples(key)
+            Segmentation.Manual().make(key)
         elif key['segmentation_method'] in [2, 6]:  # nmf and nmf-patches
-            Segmentation.CNMF()._make_tuples(key)
+            Segmentation.CNMF().make(key)
```

Comment:
you did not change the stuff inside CNMF here (and the allow _direct insert in Fluorescence)

Reply:
Updated